src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -117,5 +117,13 @@ struct EvolvedFieldsFromCharacteristicFields : db::SimpleTag {
     return "EvolvedFieldsFromCharacteristicFields";
   }
 };
+
+template <size_t Dim, typename Frame>
+struct UnitNormalOneForm : db::SimpleTag {
```

Comment:
If you really end up needing this (see comment above), this should be private to the flux struct. We have a `Normalized<UnnormalizedFaceNormal>` Tag for general use.